### PR TITLE
[zh] Sync CVE links #13643 to Chinese version

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync below updates into Chinese version:
- #13643

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/security/authentication/authn-policy/index.md [hunk 1]
```diff
@@ -504,6 +508,33 @@ spec:
 EOF
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
```
Comment (hanxiaop) [inline content/zh/docs/tasks/security/authentication/authn-policy/index.md:513]: I think this contains some dup work compared to https://github.com/istio/istio.io/pull/13646

Comment (wilsonwu) [reply content/zh/docs/tasks/security/authentication/authn-policy/index.md:513]: Yes, I will merge resolve all after the #13646 finished.

**Comment by windsonsea** [comment]
> Sync below updates into Chinese version:
> 
> * [update links for cves #13643](https://github.com/istio/istio.io/pull/13643)
> * [Add gateway-api instructions to authn policy task #13625](https://github.com/istio/istio.io/pull/13625)
> * [update links for cves #13643](https://github.com/istio/istio.io/pull/13643)

😅 
In the current desc, list 1 is duplicated with list 3, and list 2 may cause conflict.

**Comment by wilsonwu** [comment]
> > Sync below updates into Chinese version:
> > 
> > * [update links for cves #13643](https://github.com/istio/istio.io/pull/13643)
> > * [Add gateway-api instructions to authn policy task #13625](https://github.com/istio/istio.io/pull/13625)
> > * [update links for cves #13643](https://github.com/istio/istio.io/pull/13643)
> 
> 😅 In the current desc, list 1 is duplicated with list 3, and list 2 may cause conflict.

No problem, just merge the dup one, I will solve all later.

**Comment by wilsonwu** [comment]
> > > Sync below updates into Chinese version:
> > > 
> > > * [update links for cves #13643](https://github.com/istio/istio.io/pull/13643)
> > > * [Add gateway-api instructions to authn policy task #13625](https://github.com/istio/istio.io/pull/13625)
> > > * [update links for cves #13643](https://github.com/istio/istio.io/pull/13643)
> > 
> > 
> > 😅 In the current desc, list 1 is duplicated with list 3, and list 2 may cause conflict.
> 
> No problem, just merge the dup one, I will solve all later.

All Done.

**Review by windsonsea — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**